### DISCUSSION TITLE
source-files-FRITZ.Box_7530_AX-brcma9-07.31.tar.gz

### DISCUSSION THREAD

**Discussion by freetz-ng-mod** [Freetz-NG/freetz-ng#535] [Allgemein]
http://osp.avm.de/fritzbox/fritzbox-7530-ax/
bringt das was wegen des Fehlers mit BB 1.34

Für die 4060 gibt es auch was Neues
http://osp.avm.de/fritzbox/fritzbox-4060/

Bei der 6660 gibt es sogar was für die FW 7.39
http://osp.avm.de/fritzbox/fritzbox-6660-cable/

**Comment by fda77** [comment]
Diese Quellcodepakete fressen immer viel Zeit. 7530ax hat schon  07.29, das dürfte keinen grossen Unterschied machen. Ich hab eh vor demnächst alle (neuen?) kernel-source auf diffs umzustellen, damit das jeder einfacher machen kann. Ich glaub von der 6660 labor ist schon was drin, das müsste die mit falscher .config sein

**Comment by freetz-ng-mod** [comment]
Ok
